ipn/ipnstate/ipnstate.go

```diff
@@ -67,6 +67,16 @@ type Status struct {
 	User map[tailcfg.UserID]tailcfg.UserProfile
 }
 
+type NetworkLockStatus struct {
+	Enabled bool
+
+	// Head describes the AUM hash of the leaf AUM.
+	Head *[32]byte
+
+	// Pubkey describes the nodes' network-lock public key.
+	Pubkey key.NLPublic
```

Comment:
elsewhere we spell this out as PublicKey

Reply:
Updated.